### ISSUE TITLE
Use URLSearchParams to parse query params instead of using regexes

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#1075]
## Problem

We currently parse URL search params by hand using a regex in this module: https://github.com/WordPress/openverse-frontend/blob/aa2af2086e01407f5a3a9a36b05c9af6172ecda8/src/utils/get-parameter-by-name.js#L1

We should use the built in `URLSearchParams` which has support for this already: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by sarayourfriend** [comment]
@zackkrida just to make sure, was there a reason URLSearchParams wasn't used?

**Comment by zackkrida** [comment]
Yes actually @sarayourfriend , IE11 support, which we ultimately dropped, and then never revisited this.

**Comment by sarayourfriend** [comment]
Makes sense! Do you know if there are any other compromises we made for IE11 that we could revisit?

**Comment by zackkrida** [comment]
@sarayourfriend not really, I wouldn't have even thought of this one without you sparking the thought.
